cmd/boost/wallet_cmd.go

```diff
@@ -198,6 +199,19 @@ var walletList = &cli.Command{
 						wallet[marketLockedKey] = marketLockedValue
 					}
 				}
+				dcap, err := api.StateVerifiedClientStatus(ctx, addr, types.EmptyTSK)
+				if err == nil {
+					wallet[dataCapKey] = dcap
+					if !cctx.Bool("json") && dcap == nil {
+						wallet[dataCapKey] = "X"
```

Comment:
Is is possible for dcap to be nil when err is not nil?

Reply:
There are 2 scenarios here.
1. Address does exists on chain. This would result in a error for the query.
2. Address is on chain but there is no data cap assigned to that address yet or it is now over.  This scenario could be dcap to be nil when err is not nil.

Reply:
I guess I would expect that dcap is zero if there is no data cap assigned to that address... but if it's nil then this logic LGTM 👍